packages/fiori/src/TimelineItem.ts

```diff
@@ -1,13 +1,25 @@
 import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
 import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
-import event from "@ui5/webcomponents-base/dist/decorators/event-strict.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
```

Comment:
Why did you change this?

Reply:
hm, probably happened when merging conflicts, will revert it